### PR TITLE
fix: tabs and back button

### LINKED ISSUE #1144: Tabs should disappear inside detail pages (replaced by the back arrow + header)

### PR DESCRIPTION
**Pull request by cazala**
Fixes #1144

### CODE REVIEW

File: src/modules/location/reducer.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+export type LocationReducerAction = LocationChangeAction
+
+export function locationReducer(state = INITIAL_STATE, action: LocationReducerAction) {
+  switch (action.type) {
+    case LOCATION_CHANGE: {
+      return {
+        ...state,
+        hasHistory: !action.payload.isFirstRendering
```
Comment (nicosantangelo) [inline src/modules/location/reducer.ts:18]: Did not know this existed. Could actually be useful for the Mkt

**Comment by nicosantangelo** [comment]
It works! but I think I found a bug?

[Uploading Screen Recording 2020-10-28 at 13.22.36.zip…]()

**Review by nicosantangelo — COMMENTED** [review]
LGTM but I think I found a bug?

[Screen Recording 2020-10-28 at 13.22.36.zip](https://github.com/decentraland/builder/files/5453342/Screen.Recording.2020-10-28.at.13.22.36.zip)

**Comment by cazala** [comment]
good catch! fixed now :)

**Review by nicosantangelo — APPROVED** [review]

**Pull request merged**